services/venice-controller/src/main/java/com/linkedin/venice/pushmonitor/AbstractPushMonitor.java

```diff
@@ -1221,7 +1221,8 @@ private void updateStoreVersionStatus(String storeName, int versionNumber, Versi
            */
           Set<String> targetRegions = RegionUtils.parseRegionsFilterList(version.getTargetSwapRegion());
           boolean isSequentialRollForward = StringUtils.isNotEmpty(sequentialRollForwardFirstRegion)
-              && sequentialRollForwardFirstRegion.equals(regionName);
+              && sequentialRollForwardFirstRegion.equals(regionName) && version.isVersionSwapDeferred()
```

Comment:
can you add some comment as to why we needed it and later after ramp we can simply it